Sources/SurveyMakito/Views/Survey/SurveyNavigationFooterView.swift

```diff
@@ -42,6 +42,10 @@ struct SurveyNavigationFooterView: View {
              */
             if index == questions.count - 1 {
                 Button(action: {
+                    if questions[index].uid !=  response.questionId || response.values.isEmpty{
```

Comment:
Use the data config from the survey question, you can use the key-value pair allowskip: true/false. Then, check if a response has been given. It would be easier to test this if it were a method or computed property on the response model. You can find some unit tests for this in the test directory.